cmd/subctl/join.go

```diff
@@ -113,7 +113,7 @@ func addJoinFlags(cmd *cobra.Command) {
 		"enable Submariner pod debugging (verbose logging in the deployed pods)")
 	cmd.Flags().BoolVar(&joinFlags.OperatorDebug, "operator-debug", false, "enable operator debugging (verbose logging)")
 	cmd.Flags().BoolVar(&labelGateway, "label-gateway", true, "label gateways if necessary")
-	cmd.Flags().StringVar(&joinFlags.CableDriver, "cable-driver", "", "cable driver implementation")
+	cmd.Flags().StringVar(&joinFlags.CableDriver, "cable-driver", "libreswan", "cable driver implementation")
```

Comment:
We can compromise by mentioning the default in our help text instead of actually setting it as the default value. To the end user the effect is the same.

```suggestion
	cmd.Flags().StringVar(&joinFlags.CableDriver, "cable-driver", "", "cable driver implementation (default "libreswan")")
```

Reply:
That works for me! No behaviour change in the code, and the user still has the important information.

Reply:
Ya. Thats a good idea.